source/k64f_emac.c

```diff
@@ -583,7 +583,7 @@ static err_t k64f_low_level_output(struct netif *netif, struct pbuf *p)
       break;
   if (q != NULL) {
     // Allocate properly aligned buffer
-    psend = (uint8_t*)malloc(p->tot_len);
+    psend = (uint8_t*)pbuf_alloc(PBUF_RAW,p->tot_len,PBUF_POOL);
```

Comment:
`pbuf_alloc` is guarded by `SYSTEM_PROTECT`/`SYSTEM_UNPROTECT`. 

https://github.com/ARMmbed/sal-stack-lwip/blob/master/source/lwip/core/memp.c#L405

That was set up to be our C critical section API in https://github.com/ARMmbed/sal-stack-lwip/pull/54.

memp_malloc includes an alignment assert: https://github.com/ARMmbed/sal-stack-lwip/blob/master/source/lwip/core/memp.c#L420

This does not currently provide alignment guarantees beyond 4-byte alignment. What is the required alignment?